assets/js/modules/analytics/datastore/report.test.js

```diff
@@ -135,33 +157,67 @@ describe( 'modules/analytics report', () => {
 
 				muteConsole( 'error' ); // fetch will trigger 400 error.
 				registry.select( STORE_NAME ).getReport( nonAdsenseOptions );
+				await untilResolved( registry, STORE_NAME ).getReport( nonAdsenseOptions );
+
 				expect( fetchMock ).toHaveFetchedTimes( 1 );
 
+				expect( registry.select( STORE_NAME ).getAdsenseLinked() ).toBe( true );
+			} );
+
+			it( 'sets adsenseLinked to true if a success response is returned for AdSense metrics', async () => {
+				const nonAdsenseOptions = {
+					dateRange: 'last-28-days',
+					metrics: 'ga:adsenseRevenue',
+				};
```

Comment:
Probably just an oversight in the variable name here but these _are_ `AdsenseOptions`, no?